### ISSUE TITLE
Timelapse

### ISSUE THREAD

**Issue by mathishuettl** [pascalbaljetmedia/laravel-ffmpeg#180]
Hey Guys,

what is the code to exec this command with this package?

`ffmpeg -i "%05d.jpg" timelapse.mp4`

Thanks in advance!

**Comment by pascalbaljet** [comment]
I've rewritten the package and support for exporting to S3 (and other external disks) is included. Could you test the `v7` branch? Almost no breaking changes except for some filter stuff in HLS exports.

https://github.com/pascalbaljetmedia/laravel-ffmpeg/tree/v7

`composer require "pbmedia/laravel-ffmpeg:v7.x-dev"`

```php
<?php

FFMpeg::open('%05d.jpg')
    ->export()
    ->asTimelapseWithFramerate(1)
    ->inFormat(new X264)
    ->save('timelapse.mp4');
```

**Comment by pascalbaljet** [comment]
Fixed in `v7.0`
